packages/react/src/components/Button/Button.tsx

```diff
@@ -70,6 +72,8 @@ const Button = (
       className,
     )}
     type={type}
+    aria-disabled={disabled}
+    onClick={disabled ? undefined : onClick}
```

Comment:
I suggest removing this and just sticking to updating the css selectors. This might be confusing for anyone passing in `disabled` and having something else applied instead. Suggested behaviour here should be documented instead (such as developer has to make sure to disable `onClick` if they are using `aria-disabled`). We want our `Button` to behave as native as possible.

Reply:
Ahh, ok, I thought we were aiming to create something where the users of the design system could use it straight forward and know that it will simply be accessible out of the box, but I agree this could be confusing for those that do want to use the disabled functionality as-is 😅
I'll fix it when I get back from DevOps days later this week 😊